AssetRipper.TextureDecoder.Tests/Formats/ColorRGB32HalfTests.g.cs

```diff
@@ -0,0 +1,155 @@
+//This code is source generated. Do not edit manually.
+
+using AssetRipper.TextureDecoder.Rgb.Formats;
+using System.Runtime.CompilerServices;
+
+namespace AssetRipper.TextureDecoder.Tests.Formats;
+
+public partial class ColorRGB32HalfTests
```

Comment:
This class is incomplete. Some changes to the source gen were made since your pull request. You'll need to rebase and rerun the source generation.